search.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
```

Comment:
Don't we need license here?

Reply:
In accordance with the [Gentoo Social Contract](https://www.gentoo.org/get-started/philosophy/social-contract.html), I release these scripts under the terms of the GPL-2.0-or-later.